stdlib/importlib/resources.pyi

```diff
@@ -19,5 +19,8 @@ if sys.version_info >= (3, 7):
     def contents(package: Package) -> Iterator[str]: ...
 
 if sys.version_info >= (3, 9):
+    from contextlib import AbstractContextManager
     from importlib.abc import Traversable
+
```

Comment:
Weird, wonder why the extra line is fine above for black, but not here? Will fix.